contrib/ucx_perftest.service

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=UCX Perftest running in agent mode
+Requires=multi-user.target
+After=multi-user.target
+ 
+[Service]
+Type=simple
+Environment=UCX_KEEPALIVE_INTERVAL=2
+Environment=UCX_DC_MLX5_TIMEOUT=5
+Environment=UCX_RC_VERBS_TIMEOUT=5
+Environment=UCX_RC_MLX5_TIMEOUT=5
+Environment=UCX_UD_VERBS_TIMEOUT=5
+Environment=UCX_UD_MLX5_TIMEOUT=5
+Environment=UCX_RDMA_CM_TIMEOUT=5
```

Comment:
why do we need these vars?

Reply:
when running perftest with `-e`, we rely on the transport to call the error handler and terminate. using shorter timeout helps emulate more synchronuous behavior. else when perftest client ctrl-c/aborts, it can be tens for seconds before server stops, restarts and becomes available again.

Reply:
why do we need to run perftest with -e?

Reply:
without `-e` perftest never terminates if something went wrong during the test (client abort/interrupt...), so running it as a service would not work.

Reply:
i think we should not run it as a service

Reply:
ok, for that aspect I'll let Alexey comment

Reply:
Daemonizing the server side allows orchestrating client-server testing in CI. 
Running a background process in Azure is not allowed otherwise. 
We successfully use the same approach with SparkDPU.

Reply:
when we run io_demo, for example, we don't use a system service, but we run a process in the background. is that a wrong approach?

Reply:
I see that "io_demo" uses SSH to run commands on nodes. While this is a valid solution, it's more complex and error-prone.
Laying out the test flow in minimalistic Yaml steps results in straightforward code and separate logs, both are readable and make troubleshooting easier.
Simplicity and manageability are the keys to sustainable devops.

Reply:
Removed "TIMEOUT" vars.
The server side now runs on Docker.